modules/webapp/src/main/elm/Messages/Data/Language.elm

```diff
@@ -141,6 +144,9 @@ de lang =
         Japanese ->
             "Japanisch"
 
+        JpnVert ->
+            "JpnSenk"
```

Comment:
This could be "Japanisch Vertikal" I guess.

Reply:
I tried to keep the textbox roughly the same size based on the dictionary definitions but not very familiar with EMEA languages, so whatever you'd like to go with is good with me.

Reply:
Ah I see, good point! TBH I don't care so much :-) Then maybe "Jpn Vert" is a good compromise. For me, "Vert" would be better recognizable as "vertikal" than "Senk" for "senkrecht" - but I guess both are ok.

Reply:
Okidoki! We're both just trusting each other's knowledge of these different languages :D 

Will fix after the rebase.